src/pysdmx/model/__base.py

```diff
@@ -241,6 +241,20 @@ def __post_init__(self) -> None:
                 "Maintainable artefacts must reference an agency.",
             )
 
+    def short_urn(self) -> str:
```

Comment:
We need to ensure therefore that all model classes have the same name as it is specified in the URN elements. As far as I could check this will work fine.

Reply:
Thanks, @javihern98. I checked all class names again and, indeed, they are 100% in line with the SDMX Information Model 😊